cvs -z3 diff ./samples/newgrid ./src/generic/grid.cpp ./include/wx/generic/grid.h 
Index: ./src/generic/grid.cpp
===================================================================
RCS file: /home/wxcvs/wxWindows/src/generic/grid.cpp,v
retrieving revision 1.175
diff -r1.175 grid.cpp
3746c3746
<     m_extraHeight = 50;
---
>     m_extraHeight = 0;
3839,3840c3839,3840
<     int w = m_numCols > 0 ? GetColRight(m_numCols - 1) + m_extraWidth + 1 : 0;
<     int h = m_numRows > 0 ? GetRowBottom(m_numRows - 1) + m_extraHeight + 1 : 0;
---
>     int w = m_numCols > 0 ? GetColRight(m_numCols - 1) + m_extraWidth : 0;
>     int h = m_numRows > 0 ? GetRowBottom(m_numRows - 1) + m_extraHeight : 0;
8211a8212,8221
> // ----------------------------------------------------------------------------
> // Extra grid margin size
> // ----------------------------------------------------------------------------
> void wxGrid::SetMargins(int extraWidth, int extraHeight)
> {
>     m_extraWidth = extraWidth;
>     m_extraHeight = extraHeight;
>     
>     CalcDimensions();
> }
Index: ./include/wx/generic/grid.h
===================================================================
RCS file: /home/wxcvs/wxWindows/include/wx/generic/grid.h,v
retrieving revision 1.99
diff -r1.99 grid.h
1413,1417c1413
<     void SetMargins(int extraWidth, int extraHeight)
<     {
<         m_extraWidth = extraWidth;
<         m_extraHeight = extraHeight;
<     }
---
>     void SetMargins(int extraWidth, int extraHeight);
cvs server: Diffing ./samples/newgrid
Index: ./samples/newgrid/griddemo.cpp
===================================================================
RCS file: /home/wxcvs/wxWindows/samples/newgrid/griddemo.cpp,v
retrieving revision 1.51
diff -r1.51 griddemo.cpp
298a299,300
>     grid->SetMargins(50, 50);
> 


 	  	 
